bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/LedgerEntry.java

```diff
@@ -55,15 +60,28 @@
     long getLength();
 
     /**
-     * Returns the content of the entry.
+     * Returns the content of the entry into a byte array.
      *
      * @return the content of the entry
      */
     byte[] getEntry();
```

Comment:
Maybe rename to getEntryBytes()

Reply:
For backward compat, we may not able to change this?

Reply:
There shouldn't be backward compat issues with anything under the api package. client LedgerEntry doesn't inherit from this.